### PR TITLE
drii-cavalcanti/c03-docker02: creating apache image using docker

### PR DESCRIPTION
**Pull request by drii-cavalcanti**
# Description

Exercise (e.g. C01-E01): c03-docker02

Student Name: Adriana Cavalcanti

> A brief explanation of what has been done.
creating apache image using docker
## Type of change

  - [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-e03/drii-cavalcanti/c01-e03.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Create a new EC2 Instance inside the default VPC with an public IP address and connect to it though SSH:
```
Comment (kikobr82) [inline classes/01class/exercises/c01-e03/drii-cavalcanti/c01-e03.txt]: this file needs to be removed from the PR.
Don't remove it from the repo, just from the PR...

File: classes/03class/exercises/c03-docker02/drii-cavalcanti/Dockerfile.v2 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM httpd:2.4.41-alpine
+
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker02/drii-cavalcanti/Dockerfile.v2]: the contents of the /usr/local/apache2/htdocs needs to be removed on this image

Comment (kikobr82) [reply classes/03class/exercises/c03-docker02/drii-cavalcanti/Dockerfile.v2]: still missing this part... :)

**Comment by drii-cavalcanti** [comment]
> one missing point!
> let me know if need any help on that

Hi Kiko, could you be more specific? I understood that part 2 should have the folder '/usr/local/apache2/htdocs/' empty

**Comment by kikobr82** [comment]
> > one missing point!
> > let me know if need any help on that
> 
> Hi Kiko, could you be more specific? I understood that part 2 should have the folder '/usr/local/apache2/htdocs/' empty

Yeah.. and this folder have files from the base image you use in you dockerfile.
you need to remove all files.

**Pull request closed without merging**